lms/static/sass/elements/_program-card.scss

```diff
@@ -108,52 +108,79 @@
     line-height: 1;
   }
 
-  .certificate-status {
-    margin-bottom: 0;
+  .status-text {
+    display: flex;
+    margin-bottom: 5px;
+  }
 
-    .status-text {
-      font-size: font-size(x-small);
-      color: palette(grayscale, dark);
-      line-height: 1;
-    }
+  .number-status {
+    text-align: center;
+    width: 100%;
+    float: left;
+    padding-left: 5px;
+    padding-right: 5px;
+    margin-top: -8px;
+    padding-bottom: 8px;
+    font-size: 1em;
+    font-family: "Open Sans";
+  }
 
-    .secondary {
-      @extend %hide-until-focus;
-    }
+  .number-circle {
+    padding-top: 1px;
+    border-radius: 50%;
+    margin-left: auto;
+    margin-right: auto;
+    width: 23px;
+    height: 23px;
+    color: white;
+    text-align: center;
+    font-size: 0.9375em;
+    font-family: "Open Sans";
```

Comment:
I think you can use the variable ```$f-sans-serif``` for that https://github.com/edx/edx-platform/blob/master/lms/static/sass/partials/base/_variables.scss#L339